Gemfile

```diff
@@ -3,4 +3,9 @@ source 'https://rubygems.org'
 # Specify your gem's dependencies in mini_mime.gemspec
 gemspec
 
-gem "mime-types", "~> 3" if RUBY_VERSION > '2'
+gem 'mime-types', '~> 3' if RUBY_VERSION > '2'
+gem 'memory_profiler' if RUBY_VERSION >= '2.1.0'
+gem 'benchmark-ips'
+if RUBY_PLATFORM != 'java' && RUBY_VERSION > '1.9'
+  gem 'micro_mime', :git => 'https://github.com/stereobooster/micro_mime.git'
+end
```

Comment:
Can we do these with Bundler `platform: …` directives instead of RUBY_VERSION checks? It's a bit more tedious, but otherwise we have to re-bundle when we change the Ruby version we're developing with.